extensions/awt/runtime/src/main/java/io/quarkus/awt/runtime/graal/AwtFeature.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.awt.runtime.graal;
+
+import org.graalvm.nativeimage.hosted.Feature;
+import org.graalvm.nativeimage.hosted.RuntimeJNIAccess;
+
+import io.quarkus.runtime.util.JavaVersionUtil;
+
+public class AwtFeature implements Feature {
+    @Override
+    public void afterRegistration(AfterRegistrationAccess access) {
+        // Added for JDK 19+ due to: https://github.com/openjdk/jdk20/commit/9bc023220 calling FontUtilities
+        if (JavaVersionUtil.isJava19OrHigher()) {
+            try {
+                RuntimeJNIAccess.register(Class.forName("sun.font.FontUtilities"));
+            } catch (ClassNotFoundException e) {
+                throw new RuntimeException(e);
+            }
+        }
+    }
```

Comment:
Question: we are still supporting GraalVM/Mandrel for Java 17 in Quarkus 3.17+?

I thought we had all sorts of issues with it?

Reply:
We don't support it (as in we will probably not go to great extend to fix something not working there), but we also don't forbid Quarkus to use it, see https://github.com/quarkusio/quarkus/pull/39866. As long as it's not getting in the way we prefer to stay somewhat compatible with it for the time being.

Reply:
OK. FWIW I'm in favor of dropping it whenever you feel comfortable.